lib/skywalking/tracer.lua

```diff
@@ -51,6 +51,11 @@ function Tracer:start(upstream_name, correlation)
     else
         entrySpan = TC.createEntrySpan(tracingContext, req_uri, nil, contextCarrier)
     end
+
+    if (Util.checkIgnoreSuffix(req_uri, ignore_suffix)) then
+        tracingContext.is_noop = true
+    end
```

Comment:
Can we move it to the beginning of the method? We can return `TC.newNoOP()` directly if ignored.

Reply:
ok